graylog2-server/src/main/java/org/graylog/plugins/pipelineprocessor/processors/PipelineInterpreter.java

```diff
@@ -173,7 +160,7 @@ public Messages process(Messages messages, InterpreterListener interpreterListen
         return new MessageCollection(fullyProcessed);
     }
 
-    private void potentiallyDropFilteredMessage(Message message) {
+    public void potentiallyDropFilteredMessage(Message message) {
```

Comment:
Are the method visibility changes intended and needed? I don't see any comments on the change for this method and `processForResolvedPipelines`.

Reply:
Thanks for checking on these. They are intentionally done. They are used in the `IlluminateMessageProcessor` (see below). I've added clarifying comments in https://github.com/Graylog2/graylog2-server/pull/11298/commits/6f01a74bea19444b8d7e4e1183acb33327c673b5
https://github.com/Graylog2/graylog-plugin-enterprise/blob/2cf873da7612f70c4c2ca2cc9f996a7739ccb8be/enterprise/src/main/java/org/graylog/plugins/illuminate/processing/IlluminateMessageProcessor.java#L108

https://github.com/Graylog2/graylog-plugin-enterprise/blob/2cf873da7612f70c4c2ca2cc9f996a7739ccb8be/enterprise/src/main/java/org/graylog/plugins/illuminate/processing/IlluminateMessageProcessor.java#L108

Reply:
Alright :+1: